src/main/kotlin/com/cognifide/gradle/aem/common/Formats.kt

```diff
@@ -56,15 +58,19 @@ object Formats {
         return Base64.getEncoder().encodeToString(value.toByteArray())
     }
 
+    fun size(file: File): String {
+        return bytesToHuman(when {
+            file.exists() -> FileUtils.sizeOf(file)
+            else -> 0L
+        })
+    }
+
     fun bytesToHuman(bytes: Long): String {
```

Comment:
there is apache commons util for that, but it is rounding gigabytes which I don't want to have